### PR TITLE
Create build pipeline

### PR DESCRIPTION
**Pull request by djbingham**
https://eaflood.atlassian.net/browse/PSD-389

### CODE REVIEW

**Comment by djbingham** [comment]
Parent images published by this PR have been tested with the [payment-core demo service](https://github.com/DEFRA/ffc-demo-payment-service-core/pull/37).

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            aws --region $awsRegion \
+              ecr batch-delete-image \
+              --image-ids imageTag=$mergedPrImageTag \
+              --repository-name $imageNameProduction
+          """
+        }
+      }
+      stage('Tag release in github') {
+        withCredentials([
+          string(credentialsId: 'github_ffc_platform_repo', variable: 'gitToken')
```
Comment (johnwatson484) [inline Jenkinsfile]: Should use the `github-auth-token` credential.  The value contained is the same but the other version will be removed as part of the cleanup work.

**Review by johnwatson484 — COMMENTED** [review]
Looks good, just need to use the updated credential Id.

File: Jenkinsfile (hunk 1 above)
Comment (djbingham) [reply Jenkinsfile]: Thanks, updated.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            aws --region $awsRegion \
+              ecr batch-delete-image \
+              --image-ids imageTag=$mergedPrImageTag \
+              --repository-name $imageNameProduction
+          """
+        }
+      }
+      stage('Tag release in github') {
+        withCredentials([
+          string(credentialsId: 'github_ffc_platform_repo', variable: 'github-auth-token')
```
Comment (johnwatson484) [inline Jenkinsfile]: You've updated it the wrong way, `github-auth-token` should be the credential Id.

**Review by johnwatson484 — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (djbingham) [reply Jenkinsfile]: 🤦‍♂ Yep, changed it again.

**Review by johnwatson484 — APPROVED** [review]
LGTM

**Pull request merged**